feature-group-definitions/grid-animation.yml

```diff
@@ -0,0 +1,11 @@
+spec: https://drafts.csswg.org/css-grid-2/
+# status:
+#   is_baseline: false # waiting on next Safari major version
+#   support:
+#     chrome: "107"
+#     edge: "107"
+#     firefox: "76"
+#     safari: "16.1"
```

Comment:
I commented out the baseline status, since I was skeptical we'd want to allowlist this just yet—it's one of those "will be baseline soon, but not yet" features (pending Safari 17, I suppose). I did the work though, so I thought I'd show what _would_ be.